src/components/TermandPrivacy/TermOfUse.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { RegulationsCards } from './RegulationsCards';
+
+export const TermOfUse = () => {
```

Comment:
Why didn't You use an simplified version? () => (

) ?
And where is FC type for this functional component?